R/use_import_from.R

```diff
@@ -0,0 +1,69 @@
+#' Import a function from another package
+#'
+#' `use_import_from()` imports a function from another package by adding the
+#' Roxygen tag `@importFrom` to the package-level documentation (possibly
+#' created with [`use_package_doc()`]). Importing a function from another
+#' package allows you to refer to it without a namespace (e.g., `fun()` instead
+#' of `package::fun()`). Use such imports judiciously, as they make it hard to
+#' tell where a function comes from.
+#'
+#' @param package Name of package to import `fun` from
+#' @param fun Name of function to import
```

Comment:
```suggestion
#' @param package Package name
#' @param fun Function name
```

Do these (should these?) allow vectors?

Reply:
They don't currently (sort of modeled after use_package()) but I think `fun` should since it's common to have `@importFrom pkg x y z`